app/Http/Models/User.php

```diff
@@ -17,6 +17,10 @@ class User extends Model
 
 	protected $table = 'users';
 
+	protected $fillable = [
+        'name', 'address', 'age', 'avatar_path'
+    ];
```

Comment:
Please fix this indent

Reply:
In my sublime text, indent looks good @@